python/F3DPythonBindings.cxx

```diff
@@ -171,7 +176,22 @@ PYBIND11_MODULE(pyf3d, module)
         }
         catch (const f3d::options::incompatible_exception&)
         {
-          throw py::attribute_error(key);
+          if (std::holds_alternative<std::string>(value))
+          {
+            try
+            {
+
```

Comment:
empty line

Reply:
somehow got past the style checker, interesting...